### PR TITLE
Register Fullstory

### PR DESCRIPTION
**Pull request by rvadera12**
_This PR registers the FullStory destination._

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

Testing not needed as we are just registering a new destination. 

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [ ] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/destination-actions/src/destinations/index.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ register('6238cec53a46dd187d094eb7', './criteo-audiences')
 register('5f7dd78fe27ce7ff2b8bfa37', './customerio')
 register('61806e472cd47ea1104885fc', './facebook-conversions-api')
 register('61dde0dc77eb0db0392649d3', './friendbuy')
+register('62d9aa9899b06480f83e8a66', './fullstory')
```
Comment (rhall-twilio) [inline packages/destination-actions/src/destinations/index.ts:36]: I see a different ID: https://app.segment.build/partner-portal/integration/actions-fullstory/profile

Am I looking at the wrong place?

Comment (rvadera12) [reply packages/destination-actions/src/destinations/index.ts:36]: @rhall-twilio That is for the fullstory browser destination, this the cloud mode destination: 

https://app.segment.com/partner-portal/integration/actions-fullstory-cloud/profile

image

Comment (brennan) [reply packages/destination-actions/src/destinations/index.ts:36]: Looks like the id is for this destination: https://app.segment.com/partner-portal/integration/actions-fullstory-cloud/profile

**Review by brennan — APPROVED** [review]

File: packages/destination-actions/src/destinations/index.ts (hunk 1 above)
Comment (rhall-twilio) [reply packages/destination-actions/src/destinations/index.ts:36]: Ah okay. It's missing from `staging` not sure if that's expected.

**Comment by brennan** [comment]
@rvadera12 Do we also need to register new destinations here: https://github.com/segmentio/integrations/blob/32696a5a2a3a846e53a98f2a2e0f73b936af2c4d/integrations/index.js#L194-L197

**Pull request merged**